android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -454,9 +455,9 @@ public boolean containsGroupId(String groupId) {
     @Override
     public int getCount() {
       int size = mPwPairs.size();
-      if (!Utils.getMdnsEnabled(getActivity())) {
-        size -= mNumberOfHideableResults;
-      }
+      // if (!Utils.getMdnsEnabled(getActivity())) {
```

Comment:
before we were toggling showing and not showing the folder. now its controlled by whether you have any of the mdns/wifi scanners enabled